### PR TITLE
Fix dml buttons visibility on users view

### LINKED ISSUE #16202: Missing Show/Edit/Delete under Security -> Users in 2.1.0
**Apache Airflow version**: 2.1.0

**Browsers**: Chrome and Firefox

**What happened**:

Before upgrading to 2.1.0

before

After upgrading to 2.1.0

after

**What you expected to happen**:

Show/Edit/Delete under Security -> Users are available

**How to reproduce it**:

Go to Security -> Users (as an admin of course)

### PR DESCRIPTION
**Pull request by Walderman**
---
This PR fixes #16202 where Add/Edit/Delete buttons/permissions are missing when using OUATH, LDAP authentication providers even for admin role.
I was able to replicate the issue and tested that my changes resolve the problem (tested with Google OUATH and with LDAP).
This is my first PR, I went through all the docs, but please advise if I missed something.

@jhtimmins , could you please have a look?

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -4236,18 +4236,11 @@ class CustomViewMenuModelView(ViewMenuModelView):
     ]
 
 
-class CustomUserDBModelView(UserDBModelView):
-    """Customize permission names for FAB's builtin UserDBModelView."""
+class CustomUserCommonModelView():
```
Comment (ashb) [inline airflow/www/views.py]:
```suggestion
class CustomUserCommonModelView:
```

File: airflow/www/views.py [hunk 2]
```diff
@@ -4236,18 +4236,11 @@ class CustomViewMenuModelView(ViewMenuModelView):
...
+    """Customize permission names for FAB's builtin UserDBModelView. Common settings for all providers user views"""
```
Comment (ashb) [inline airflow/www/views.py]: Comment isn't right anymore.

**Review by ashb — CHANGES_REQUESTED** [review]
This changes makes Admin users unable to edit existing users, or even visit the List Users page.

File: airflow/www/views.py (hunk 2 above)
Comment (Walderman) [reply airflow/www/views.py]: thanks @ashb , i will update

Comment (Walderman) [reply airflow/www/views.py]: all users views for providers inherit from the common class that lists the crud permissions, so all of them will now have the add/delete/edit buttons available

**Comment by Walderman** [comment]
no longer required

**Pull request closed without merging**